packages/cli/src/cmds/preview/constants.ts

```diff
@@ -107,7 +107,9 @@ export const doctorResult = {
   NodeNotSupported: `Node.js (@CurrentVersion) is not in the supported version list (@SupportedVersions).`,
   NodeSuccess: `Supported Node.js version (@Version) is installed`,
   InstallNode:
-    "Go to https://nodejs.org/about/releases/ to install Node.js (recommended version v14)",
+    "Go to https://nodejs.org/about/releases/ to install Node.js (recommended version v16)",
```

Comment:
version and v are duplicate, use v16 only.

Reply:
Added.